### DISCUSSION TITLE
🔥 Ant Design 社区 T 恤创意征集

### DISCUSSION THREAD

**Discussion by afc163** [ant-design/ant-design#42772] [Ideas]
1. 建议使用 Ant Design Logo / 主色 / 组件等设计元素
2. 程序员向、科技向、开源向、前端向
3. 积极向上，美观大方，适合外出
4. 合法合规，杜绝抄袭，无知识产权问题
5. 参与方式：在此讨论下贴图回复，截止 2023 年 06 月 30 日
6. 由官方发起投票，票高者中选。若无合适的创意，则此活动流拍。
5. 统一由官方团队制作，作为社区礼物，赠送给对 antd 社区有突出贡献的成员，2023 年内有贡献的优先

> Ant Design 官方团队保留活动解释权

**Comment by RedJue** [comment]
![image](https://github.com/ant-design/ant-design/assets/21119589/fa80f220-dcbf-4f61-ba05-c6150abd3b23)
类似这种，纯白或纯黑，中间印一个大大的 antd logo。

**Comment by kiner-tang** [comment]
设计能力有限，画得有点丑，如果社区有设计大佬觉得这个思路可行，可以在这个思路的基础上优化一下，大概思路是：“今年是AntDesign开源8周年，所以增加一个元素8，把8横过来放，就是无限符号，AntDesign + ∞ ，代表依靠社区的力量，AntDesign将有无限的可能，也希望有无限的贡献者和志愿者积极参与社区贡献，让 AntDesign 越来越好”
![WechatIMG269 (1)](https://github.com/ant-design/ant-design/assets/10286961/de15234a-1e54-43fd-8d8a-09b8b1ac8d98)

**Reply by afc163** [reply]
stable diffusion 来一发试试

**Comment by MadCcc** [comment]
![image](https://github.com/ant-design/ant-design/assets/27722486/74d86d66-b389-4512-bf4d-51a7a96cf6cd)
![image](https://github.com/ant-design/ant-design/assets/27722486/912d97bc-2864-4fa6-99cf-1c2e58bd7409)

**Comment by afc163** [comment]
稿件太少，流拍。

**Reply by oushu1tanqiang1** [reply]
尬住了
